packages/react/src/components/UnorderedList/UnorderedList-test.js

```diff
@@ -8,36 +8,47 @@
 import React from 'react';
 import UnorderedList from '../UnorderedList';
 import ListItem from '../ListItem';
-import { shallow } from 'enzyme';
 import { settings } from 'carbon-components';
+import { render, screen } from '@testing-library/react';
 
 const { prefix } = settings;
+
 describe('UnorderedList', () => {
-  describe('Renders as expected', () => {
-    const list = shallow(
-      <UnorderedList className="some-class">
+  it('should render children as expected', () => {
+    render(
+      <UnorderedList>
         <ListItem>Item</ListItem>
       </UnorderedList>
     );
+    expect(screen.getByText('Item')).toBeDefined();
```

Comment:
```suggestion
    expect(screen.getByText('Item')).toBeInTheDocument();
```

Maybe this?

Reply:
@joshblack is there a reason why `toBeDefined` doesn't work here? for learning purposes.

Reply:
@jnm2377 I think it 100% works, just was trying to more closely match what I was thinking the assertion was. `toBeDefined` is a little less strict than `toBeInTheDocument` as it will allow more things to match it than `toBeInTheDocument`

Reply:
gotcha! that makes sense 👍🏽